snippets/terraform/deployments/with-diagnostic-setting/main.tf

```diff
@@ -0,0 +1,59 @@
+terraform {
+  required_version = "~> 1.3"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.85"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+module "prerequisites" {
+  source   = "../../prerequisites"
+  location = var.location
+  name     = var.name
+  tags     = var.tags
+}
+
+data "azurerm_storage_account" "example" {
+  name                = var.storage_account_name
+  resource_group_name = var.storage_account_resource_group
+}
+
+resource "azurerm_nginx_deployment" "example" {
+  name                     = var.name
+  resource_group_name      = module.prerequisites.resource_group_name
+  sku                      = var.sku
+  location                 = var.location
+  capacity                 = 50
```

Comment:
50?

Reply:
There was a conversation a long time ago about not having this be set to 20 but rather 50. I cannot recall exactly why other than to illustrate you can create deployments with >20 capacity.

Reply:
nit: I am okay with what you have here. Just want to avoid a confusion around this only working if the capacity is 50. I'd recommend lowering it as it is an example use-case and we don't want to create too many either